addons/website/views/snippets/s_image_text_overlap.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_text_overlap" name="Image - Text Overlap">
+    <section class="s_image_text_overlap pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item o_grid_item_image order-lg-0 g-col-lg-7 g-height-9 col-lg-7" style="grid-area: 1 / 1 / 10 / 8; --grid-item-padding-y: 0; z-index: 1; order: 2;">
+                    <img src="/web/image/website.s_picture_default_image" class="img img-fluid mx-auto rounded" alt=""/>
+                </div>
+                <div class="o_grid_item order-lg-0 g-col-lg-6 g-height-7 col-lg-6 o_cc o_cc5 justify-content-center rounded" style="grid-area: 2 / 7 / 9 / 13; --grid-item-padding-y: 48px; --grid-item-padding-x: 48px; z-index: 2; order: 1;">
```

Comment:
Don't forget to remove the `justify-content-center` class :wink:

Reply:
All fixed and I also changed the `<h3>` to `<h2 class="h3-fs">`